src/molecules/AudioPlayer/AudioPlayer.js

```diff
@@ -1,9 +1,7 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import $ from 'webpack-zepto';
-import Player from 'apm-html5-player';
-import FormatTime from 'apm-html5-player/dist/FormatTime';
-import AudioAnalytics from 'apm-html5-player/dist/AudioAnalytics';
+import { Player, FormatTime, AudioAnalytics } from 'apm-html5-player';
```

Comment:
should we just get rid of this file all together -  we aren't using the player form titan

Reply:
I'm happy to, if you're sure! I just wanted a bit more confirmation that it truly wasn't being used anywhere. Then again, it shouldn't be working even if it was.

Reply:
I looked in news and podcasts wasn't being used